package/fuseki/package

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# Copyright (c) 2025 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(fuseki)
+## This should be a short description targeted at users of your package
+# Start directly with a noun or a verb, do not include “for the reMarkable”
+pkgdesc="Go frontend using GnuGo"
+## URL to where the project can be found on the Internet
+url="https://github.com/palfrey/fuseki"
+## Version of the package (see below)
+pkgver=0.0.1-1
+## ISO-8601 timestamp at which the source of this version was released
+timestamp=2025-08-25T16:56+0100
+## Section that best matches this package
+# See <https://github.com/toltec-dev/toltec/blob/stable/docs/package.md#section>
+# for available sections
+section=games
+## This should be your contact information
+maintainer="Tom Parker-Shemilt <palfrey@tevps.net>"
+## A license under which the upstream source is available
+# Choose among the list of valid SPDX license identifiers
+# <https://spdx.org/licenses/>
+license=AGPL-3.0-or-later
+archs=(rm1)
+
+## Which Docker image to use for building the package
+# See <https://github.com/toltec-dev/toolchain> for available images
+# The images are debian based and allow you to install additional packages with apt.
+# Examples (version may be out of date): base:v1.1, qt:v1.1, python:v1.1, rust:v1.1
+image=rust:v3.1
+## Whitespace-separated list of source archives that are needed to build the package
+# Archives will be automatically extracted, stripping any useless containing
+# directories, and made available during the build step below
+source=(
+    https://github.com/palfrey/gnugo/archive/51f99b7a00b7acf295475b56c349c857a9aa2d0e.zip
+    https://github.com/palfrey/fuseki/archive/638191be13be06a8d7428e7739ed947904ef3e56.zip
+)
+## SHA-256 checksums of the source archives above
+sha256sums=(
+    2624c57641c7cb28699df693c6da65f6d192f9867554fcfcf830b17db851a702
+    37e4ff419a5a697039e9aa9e5b156248fcf6ed7a97867aae13aa623a7ee6a6ea
+)
+
+noextract=(638191be13be06a8d7428e7739ed947904ef3e56.zip 51f99b7a00b7acf295475b56c349c857a9aa2d0e.zip)
+
+prepare() {
+    cd build/package/fuseki/rm1/src
+    unzip -o 638191be13be06a8d7428e7739ed947904ef3e56.zip
+    unzip -o 51f99b7a00b7acf295475b56c349c857a9aa2d0e.zip
+}
+
+makedepends=("build:autoconf" "build:automake")
+
+build() {
+    cd gnugo-51f99b7a00b7acf295475b56c349c857a9aa2d0e
+    autoreconf || true
```

Comment:
What are you swallowing with the `|| true` here?

Reply:
Could you add a comment with information as to why it's being done this way, a link to automake documentation that recommends this would be fine.

Reply:
I have seen those instructions _somewhere_ but it's been some time and I can't find them again. Turned out to be easier to find the set of commands autoreconf runs (which includes the automake) and figure out the set that needed running to make autoreconf work properly.